### DISCUSSION TITLE
List of equations

### DISCUSSION THREAD

**Discussion by bernardo-cruz** [quarto-dev/quarto-cli#5345] [Q&A] [answered]
Is there a way to write a list of equations in the same way as for list of figures and list of tables in  a pdf-quarto report?

**Comment by mcanouil** [comment, marked as answer]
Because PDF means LaTeX (currently), it means any LaTeX tools/tricks works, thus you can use/adapt for example <https://tex.stackexchange.com/questions/14317/list-of-equations-table-of-equations#14328>.
You are likely to need to use partials, see <https://quarto.org/docs/journals/templates.html#latex-partials>.

**Reply by cderv** [reply]
Could be a feature that would be worth adding 🤔 Didn't look if it is easy or not to add. Just thinking feature-wise

what do you think @dragonstyle ?

**Reply by mcanouil** [reply]
If considering a feature, would it make more sense to think of a more general API to list things that are cross-referenceable, not only equations?
To note, beside HTML it can be a bit tricky.

**Reply by cderv** [reply]
Yes - possibly something that will come in the next crossref coming work for 1.4

**Reply by cscheid** [reply]
> If considering a feature, would it make more sense to think of a more general API to list things that are cross-referenceable, not only equations?

> To note, beside HTML it can be a bit tricky.

For 1.4, we can (maybe!) look at the HTML format. My goal with the crossref work is to "clean house" first, from which we can then work on features. The main pain points are inconsistency of syntax and features across different kinds of crossreferenceable artifacts, and generalizability of these across formats.

A generic list of crossreferenceable things will be quite tricky to make work on LaTeX. The stackexchange solution works as a standalone thing, but to do this right in quarto is much harder, where we expect our setup to work across a variety of latex packages, etc.
